src/main/java/bio/terra/cli/command/config/List.java

```diff
@@ -55,6 +59,8 @@ public ConfigListReturnValue(
   private static void printText(ConfigListReturnValue returnValue) {
     OUT.println("[browser] browser launch for login = " + returnValue.browser);
     OUT.println("[image] docker image id = " + returnValue.image);
+    OUT.println(
+        "[resources] max number of resources to allow per workspace = " + returnValue.resources);
```

Comment:
"allow" makes it sound like we're enforcing how many are actually there. It seems more like "max resources to display per workspace."

Reply:
This is actually enforcing how many resources are allowed for workspaces where you want to use the CLI. If there are more than that many resources, the CLI will throw an error anytime you try to do anything resource-related (e.g. `list`, `create`, ...).